lib/creek/styles/value_convert/string_converter.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Creek
+  class Styles
+    module ValueConvert
+      class StringConverter < Base
+
+        SUPPORT_TYPES = %w[str inlineStr].freeze
```

Comment:
Why it is string here and symbol in all the other files? (e. g. lib/creek/styles/value_convert/date_time_convert.rb)

Reply:
Because type is a string and style is a character
I did not change this behavior in the library